content/en/apps/payments/payments-api.md

```diff
@@ -0,0 +1,145 @@
+
+## `createPaymentAccount`
+
+Create a record of an existing payment account details in the application.
+
+### Parameters
+
+* `accountName`: String - Account name to be stored against the account details that are received from the PSP.
+
+* `currency`: FiatCurrency - The currency that the account should be created with.
+
+* `psp`: String - The PSP to create the account with.
+
+* `agent`: Party [Optional] - the Agent which runs the Payment Service for operating on the account. If not provided, the account will default to an agent configured via the node's Payment Service CorDapp config.
+
+* `details`: Map<String, String> - [OPTIONAL] additional Key-value data associated with the Payment Account. These details will be stored in the underlying PaymentAccountDetailsState shared between agent and node - but are NOT available through the returned PaymentAccount object.
+
+### Return Type
+
+`AccountDetails` - The payment account details.
+
+## `updatePaymentAccount`
+
+Send a request to the given PSP to update the underlying `PaymentAccountDetailsState` with the provided details.
+
+### Parameters
+
+details: Map<String, String> - additional Key-value data associated with the Payment Account. These details are stored in the underlying `PaymentAccountDetailsState` shared between agent and node - but are not available through the returned `PaymentAccount` object.
+
+### Return Type
+
+`AccountDetails` - The payment account details
+
+## `paymentAccounts`
+
+Get a list of `PaymentAccount`s for all the accounts owned by the party passed as a parameter.
+
+The `AccountDetails` object captures the owning key for these details which could refer to a well known party or Corda account.
+
+#### Parameters
+
+`owningParty`: `AbstractParty` - the owning party of the payment accounts.
+
+#### Return Type
+
+`List<PaymentAccount>` - The list of internal account references. Returns an empty list if none exist.
+
+## `paymentAccountById`
+
+Get the `PaymentAccount` associated with a unique account identifier (signature).
+
+#### Parameters
+
+`accountSignature`: String - the signature matching the target's `PaymentAccount.signature`
+
+#### Return Type
+
+`PaymentAccount` - The payment account object.
+
+
+## `localTransactions`
+
+This method returns the PaymentState for all payments made on the payment service for a given account while restricting the results to local transactions - those made only through the Corda Payments Service and which were requested by this node. (For a set of all payments related to the account see [transactions].
+
+### Parameters
+
+*  `accountSignature`: String - the account reference to query the vault against
+
+*  `fromTime`: Instant - start of time-window
+
+*  `toTime`: Instant - end of time-window
+
+*  `queryCriteria`: QueryCriteria.VaultQueryCriteria? - [Optional] additional query constraints
+
+*  `paging`: PageSpecification? - [Optional]
+
+### Return Type
```

Comment:
/**
 * Records a general transaction recorded on the PSP.
 */
class PaymentTransaction